src/pages/_app.tsx

```diff
@@ -84,11 +85,17 @@ function MyApp({
           ) : (
             <BreadcrumbsAppenderAG />
           )}
-          <ContentWrapperStyled tabIndex={-1} id="maincontent">
```

Comment:
Vil ikke dette føre til forskjellig styling for AG og SM? Vil noen endringer propagere seg videre? Evt kan vi jo prøve å holde denne denne felles og flytte over styling i de respektive "audiencene".

Reply:
evt bruke page-container på begge hvis det går?

Reply:
Kan ikke bruke PageContainer på sykmeldt enda, kun laget for AG. Bendik snakket om at vi kanskje bør gjøre det, men når alle er klare.

Ellers blir det ulik styling på toppnivå for AG og SM, som jeg tror går greit. Det burde helst være felles da. Men det er fordi sidemenyen styrer en del layout

Reply:
Men er åpen for å gjøre det likere på et eller annet vis om det er mulig